tests/fixtures/dos_multiple_briefs_fixture.json

```diff
@@ -3,6 +3,7 @@
     {
       "applicationsClosedAt":"2016-03-24T00:00:00.000000Z",
       "backgroundInformation":"Here is the first requirements summary.",
+      "specialistRole":"Content Specialist",
```

Comment:
I don't think this is how `specialistRole` looks in serialized briefs.  The stored values are things like `"specialistRole": "contentDesigner"` and `"specialistRole": "businessAnalyst"`, so our test fixtures should reflect this.